orchagent/portsorch.cpp

```diff
@@ -1817,61 +1817,6 @@ void PortsOrch::initializeQueues(Port &port)
     }
 
     SWSS_LOG_INFO("Get queues for port %s", port.m_alias.c_str());
-
-    /* Create the Queue map in the Counter DB */
-    /* Add stat counters to flex_counter */
```

Comment:
do we need queue map for applying qos configuration?

Reply:
No. We use only Port::m_queue_ids vector. We use the maps only:
1. swss: pfc handler
2. swss: countercheckorch
3. queuestat
4. snmp